modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java

```diff
@@ -528,6 +528,8 @@ else if (clientCfg.getSecurityCredentialsProvider() == null)
 
         clientCfg.setPingTimeout(args.pingTimeout());
 
+        clientCfg.setConnectTimeout(args.connectionTimeout());
```

Comment:
It seems to me, we need to update `CommandHandler#printHelp()` method as well.
```
        logger.info("Default values:");
        ...
        logger.info(DOUBLE_INDENT + "PING_TIMEOUT=" + DFLT_PING_TIMEOUT);
        logger.info(DOUBLE_INDENT + "CONNECTION_TIMEOUT=" + DFLT_CONNECTION_TIMEOUT);
        ...
```

I could not find any description of these parameters, that is sad. Anyway, this issue should be addressed by a new ticket I think.

Reply:
Fixed.